docs/docs/codes/modifiers.mdx

```diff
@@ -30,13 +30,17 @@ These act like any other key code.
 
 ### Modifier Functions
 
+Modifier functions add one or more modifiers to a code.
+
 These functions take the form: `XX(code)`
 
-- They _apply_ the modifier to another _code_.
-  - e.g. `&kp LS(A)` = `LEFT_SHIFT`+`A`
-    - aka a _shifted_ (capitalized) **A**.
-  - The _code_ and the _modifier_ are handled _as one_.
-- They can be _combined_:
-  - e.g. `&kp LC(RA(B))` = `LEFT_CONTROL`+`RIGHT_ALT`+`B`
-- Some codes _include_ modifiers in their definition.
-  - e.g. `DOLLAR` = `LS(N4)`
+- Modifier functions apply a modifier to a code:
+  - `&kp LS(A)` = `LEFT_SHIFT`+`A` (a capitalized **A**).
+- They can be combined:
+  - `&kp LC(RA(B))` = `LEFT_CONTROL`+`RIGHT_ALT`+`B`
+- Some basic codes codes already include a modifier function in their definition:
```

Comment:
```suggestion
- Some basic codes already include a modifier function in their definition:
```

Reply:
@okke-formsma this PR looks good, once this suggested fix is in. Thanks!